src/constants.ts

```diff
@@ -3,3 +3,11 @@ const { homepage, version } = require('../package') as Record<string, string>;
 
 export { version };
 export const USER_AGENT = `DiscordBot (${homepage.split('#')[0]}, ${version})`;
+
+export const WSEvents = {
```

Comment:
(Screaming) snake case should be used here
```suggestion
export const WS_EVENTS = {
```

Reply:
Down